pkg/volume/volume.go

```diff
@@ -28,6 +28,32 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	Accountable
+}
+
+type VolumeDefaults struct {
```

Comment:
Do we need this indirection?  Do we really think there will be other defaults that all have the same overlap to justify a shortcut struct like this?  I'd rather see AccountingNil in each Volume than this.